deploy/k8s/install.k8s.sh

```diff
@@ -0,0 +1,523 @@
+    echo "Copy $template_file to $full_path"
+
+    if [[ -f $full_path ]] && ! confirm y "File $relative_path already exists. Would you like to replace it?"; then
+        rm -f "$template_file"
+    else 
+        mv -f "$template_file" "$full_path"
+    fi
+}
+
+# This function prompts the user for an input for a non-empty Mongo root password. 
+read_mongo_password() {
+    read -srp 'Set the mongo password: ' mongo_root_password
+    while [[ -z $mongo_root_password ]]; do
+        echo ""
+        echo ""
+        echo "+++++++++++ ERROR ++++++++++++++++++++++"
+        echo "The mongo password cannot be empty. Please input a valid password string."
+        echo "++++++++++++++++++++++++++++++++++++++++"
+        echo ""
+        read -srp 'Set the mongo password: ' mongo_root_password
+    done 
+}
+
+# This function prompts the user for an input for a non-empty Mongo username. 
+read_mongo_username() {
+    read -rp 'Set the mongo root user: ' mongo_root_user
+    while [[ -z $mongo_root_user ]]; do
+        echo ""
+        echo "+++++++++++ ERROR ++++++++++++++++++++++"
+        echo "The mongo username cannot be empty. Please input a valid username string."
+        echo "++++++++++++++++++++++++++++++++++++++++"
+        echo ""
+        read -rp 'Set the mongo root user: ' mongo_root_user
+    done
+}
+
+urlencode() {
+    # urlencode <string>
+    local old_lc_collate="$LC_COLLATE"
+    LC_COLLATE=C
+
+    local length="${#1}"
+    for (( i = 0; i < length; i++ )); do
+        local c="${1:i:1}"
+        case $c in
+            [a-zA-Z0-9.~_-]) printf "$c" ;;
+            *) printf '%%%02X' "'$c" ;;
+        esac
+    done
+
+    LC_COLLATE="$old_lc_collate"
+}
+
+generate_password() {
+    # Picked up the following method of generation from : https://gist.github.com/earthgecko/3089509
+    LC_CTYPE=C tr -dc 'a-zA-Z0-9' < /dev/urandom | fold -w 13 | head -n 1
+}
+
+confirm() {
+    local default="$1"  # Should be `y` or `n`.
+    local prompt="$2"
+
+    local options="y/N"
+    if [[ $default == y || $default == Y ]]; then
+        options="Y/n"
+    fi
+
+    local answer
+    read -rp "$prompt [$options] " answer
+    if [[ -z $answer ]]; then
+        # No answer given, the user just hit the Enter key. Take the default value as the answer.
+        answer="$default"
+    else
+        # An answer was given. This means the user didn't get to hit Enter so the cursor on the same line. Do an empty
+        # echo so the cursor moves to a new line.
+        echo
+    fi
+
+    [[ yY =~ $answer ]]
+}
+
+
+echo_contact_support() {
+    echo "Please contact <support@appsmith.com> with your OS details and version${1:-.}"
+}
+
+bye() {  # Prints a friendly good bye message and exits the script.
+    set +o errexit
+    echo "Please share your email to receive support with the installation"
+    read -rp 'Email: ' email
+    curl -s -O --location --request POST 'https://hook.integromat.com/dkwb6i52am93pi30ojeboktvj32iw0fa' \
```

Comment:
Please update all curl commands to integromat with the following:
```suggestion
    curl -s --location --request POST 'https://hook.integromat.com/dkwb6i52am93pi30ojeboktvj32iw0fa' \
```